src/App.js

```diff
@@ -0,0 +1,32 @@
+import './App.css';
+import axios from "axios";
+import Greetings from './components/greetings';
+import { useEffect, useState } from 'react';
+
+const API_URL = "http://localhost:3000/api/v1/greetings";
+
+function getAPIData() {
+  return axios.get(API_URL).then((response) => response.data)
+}
+
+function App() {
+  const [greetings, setGreetings] = useState([]);
+
+  useEffect(() => {
+    let mounted = true;
+    getAPIData().then((items) => {
```

Comment:
- [ ] as required you should create a store and action to fetch from the API so please try to use redux in order to complete the requirements 
- Screenshot of the requirements 
<img width="877" alt="Screenshot 2022-12-03 at 13 11 05" src="https://user-images.githubusercontent.com/61790587/205440384-312f3798-d63f-4fd4-b7d1-a3b99910d30e.png">